configure

```diff
@@ -50,5 +50,22 @@ fi
 
 version=`pkg-config --modversion zlib`
 echo "zlib ${version} is available ok"
-exit 0
 
+# Find R compilers
+CC=`${R_HOME}/bin/R CMD config CC`
+CFLAGS=`${R_HOME}/bin/R CMD config CFLAGS`
+CPPFLAGS=`${R_HOME}/bin/R CMD config CPPFLAGS`
+
+# Test if we have a OPENMP compatible compiler
+echo "#include <omp.h>\nint main () { return omp_get_num_threads (); }" | ${CC} ${CPPFLAGS} ${PKG_CFLAGS} ${CFLAGS} ${SHLIB_OPENMP_CFLAGS} -E -xc - >/dev/null 2>&1 || R_NO_OPENMP=1;
+
+# Write to Makevars
+if [ $R_NO_OPENMP ]; then
+  sed -e "s|@openmp_cflags@||" src/Makevars.in > src/Makevars
```

Comment:
Can we add a message here indicating openMP wasn't detected? I think we want to be quite loud to the user that they should try and install openMP, perhaps linking to the installation wiki for how to do so.

In that sense the current behavior was nice, since we prefer users to be running with openMP support & the failure would force that (just that the error was not helpful/instructive)

Reply:
Sure, do you have a suggestion on what specifically it should say?

Reply:
thanks Jim, have adjusted, will give Matt final editorial cut